### PR TITLE
MGDOBR-1001: Fix integration tests dns

### PR DESCRIPTION
**Pull request by r00ta**
[MGDOBR-1001](https://issues.redhat.com/browse/MGDOBR-1001)

## PR owner checklist

Please make sure that your PR meets the following requirements:

- [ ] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config).  
  *Run `mvn clean verify -DskipTests` so that imports are correctly set.*
- [ ] Your commit messages are clear and reference the JIRA issue e.g: "[MGDOBR-1] - $clear_explanation_of_what_you_did"
- [ ] The layers in the `kustomize` folder have been updated accordingly.
- [ ] All new functionality is tested
- [ ] Pull Request title is properly formatted: `MGDOBR-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

## PR reviewer(s) checklist

- [ ] Check if code and Github action workflows have been modified and if the modification is safe
- [ ] If the modification is safe, add the `safe to test` label

<details>
<summary>
How to trigger pipelines and use the bots:
</summary>

- <b>Run the end to end pipeline</b>  
  Annotate the pull request with the label: `safe to test`. If you want to run the pipeline again, remove and add it again.

- <b>Rebase the pull request</b>  
  Comment with: `/rebase`.
- <b>Deploy BOT</b>
  Comment with `/deploy <dev,stable> when the PR has been merged to deploy to a target environment.

</details>

### CODE REVIEW

File: integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/IngressSteps.java [hunk 1]
```diff
@@ -57,11 +66,23 @@ public void ingressOfBridgeIsNotAvailableWithinMinutes(String testBridgeName, in
...
-                .untilAsserted(() -> IngressResource.optionsJsonEmptyEventResponse(context.getManagerToken(), endpoint)
-                        .then()
-                        .statusCode(Matchers.anyOf(Matchers.is(404), Matchers.is(503))));
+                .untilAsserted(() -> {
+                    try {
+                        IngressResource.optionsJsonEmptyEventResponse(context.getManagerToken(), endpoint)
+                                .then()
+                                .statusCode(Matchers.anyOf(Matchers.is(404), Matchers.is(503)));
+                    } catch (UnknownHostException | SSLException ignored) {
+                        // The DNS was properly deleted and the record expired.
```
Comment (danielezonca) [inline integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/IngressSteps.java:83]: I would add a debug log just to improve troubleshooting if necessary

**Review by danielezonca — APPROVED** [review]
LGTM

**Review by sutaakar — APPROVED** [review]

File: integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/IngressSteps.java [hunk 2]
```diff
@@ -79,8 +83,9 @@ public void ingressOfBridgeIsNotAvailableWithinMinutes(String testBridgeName, in
                         IngressResource.optionsJsonEmptyEventResponse(context.getManagerToken(), endpoint)
                                 .then()
                                 .statusCode(Matchers.anyOf(Matchers.is(404), Matchers.is(503)));
-                    } catch (UnknownHostException | SSLException ignored) {
+                    } catch (UnknownHostException | SSLException e) {
                         // The DNS was properly deleted and the record expired.
+                        LOGGER.info("An exception '{}' was raised when trying to call the bridge ingress, considering the bridge as deleted.", e.getMessage());
```
Comment (sutaakar) [inline integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/IngressSteps.java:88]: The usual logging is unfortunately not working properly, please use:
`context.getScenario()..log(.....);`

https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/a1df45011f0475d1dd0cf19322f5e86c892cdcfb/integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/context/TestContext.java#L51

**Comment by r00ta** [comment]
Merging as the failures are not related to this PR

**Pull request merged**